projects/storefrontapp/src/environments/models/feature.model.ts

```diff
@@ -1,5 +1,5 @@
 import { ModuleWithProviders, Type } from '@angular/core';
 
 export interface FeatureEnvironment {
-  imports: Array<Type<any> | ModuleWithProviders<{}>>;
+  imports: (any[] | Type<any> | ModuleWithProviders<{}>)[];
```

Comment:
Why it was changed?

Reply:
Was needed when I had the feature imports without lazy loading (see https://github.com/SAP/spartacus/pull/8918/commits/7685410a8ca56e4749786ec918455fb0ea2c79f3), but I don't recall the exact reason. I can revert that